ddtrace/llmobs/_writer.py

```diff
@@ -507,11 +526,19 @@ def dataset_get_with_records(self, dataset_name: str, project_name: Optional[str
             has_next_page = False
             if next_cursor:
                 has_next_page = True
-                list_path = f"{list_base_path}?page[cursor]={next_cursor}"
+                url_options["page[cursor]"] = next_cursor
+                list_path = f"{list_base_path}?{urllib.parse.urlencode(url_options, safe='[]')}"
```

Comment:
should this be un-indented? will this only get set (& we only use the version filter) when we're paging results?

Reply:
hmmm yea you are right, good catch!